ingestion/src/metadata/ingestion/source/database/dbt/dbt_config.py

```diff
@@ -218,62 +214,94 @@ def _(config: DbtCloudConfig):  # pylint: disable=too-many-locals
         raise DBTConfigException(f"Error fetching dbt files from DBT Cloud: {exc}")
 
 
+def get_blobs_grouped_by_dir(blobs: List[str]) -> Dict:
```

Comment:
nit - we could type this `Dict` a bit better or even use a pydantic model. But at least we could say `Dict[str, List[str]]` or something to help understand that the key is the subdir and the value the list of dbt files

Reply:
Added the type